code/game/gamemodes/gang/dominator.dm

```diff
@@ -196,9 +211,9 @@
 		priority_announce("Network breach detected in [locname]. The [gang.name] Gang is attempting to seize control of the station!","Network Alert")
 		gang.domination()
 		SSshuttle.registerHostileEnvironment(src)
-		src.name = "[gang.name] Gang [src.name]"
+		name = "[gang.name] Gang [name]"
 		operating = 1
-		icon_state = "dominator-[gang.color]"
+		update_icon()
 
 		countdown.color = gang.color_hex
```

Comment:
Changing it to not colorize the text would be trivial if someone wants to do that